nsq/reader.py

```diff
@@ -358,7 +344,7 @@ def _maybe_update_rdy(self, conn):
         if self.backoff_timer.get_interval() or self.max_in_flight == 0:
             return
 
-        if conn.rdy <= 1 or conn.rdy < int(conn.last_rdy * 0.25):
+        if conn.rdy == 1:
```

Comment:
Perhaps it's worth a comment since it took me a little while to figure this out.